src/lightning/pytorch/loggers/neptune.py

```diff
@@ -48,6 +49,23 @@
 _INTEGRATION_VERSION_KEY = "source_code/integrations/pytorch-lightning"
 
 
+def catch_inactive(func: Callable) -> Callable:
+    """Neptune client throws `InactiveRunException` when trying to log to an inactive run.
```

Comment:
This description doesn't explain what the function does directly, so it's not really a docstring. Perhaps this explanation should simply be a comment.

Reply:
Sure. Done